src/test/java/org/sagebionetworks/bridge/exporter/integration/NotificationTest.java

```diff
@@ -175,6 +163,90 @@ public static void beforeClass() throws Exception {
             developer.getClient(SchedulesApi.class).createSchedulePlan(newPlan).execute();
         }
 
+        // Make sure we have the engagement survey.
+        List<Survey> surveyList = developer.getClient(SurveysApi.class).getPublishedSurveys().execute().body()
+                .getItems();
+        Optional<Survey> optionalEngagementSurvey = surveyList.stream()
+                .filter(s -> TEST_ENGAGEMENT_SURVEY.equals(s.getIdentifier())).findAny();
```

Comment:
Everyone uses survey identifier as a key... we should add this to the API the same way we added healthCode:AAA to the API. JIRA filed: https://sagebionetworks.jira.com/browse/BRIDGE-2259

Reply:
Similarly, the schema system currently relies on survey IDs being unique. Related JIRA https://sagebionetworks.jira.com/browse/BRIDGE-1833